StackExchange.Redis/StackExchange/Redis/ConfigurationOptions.cs

```diff
@@ -78,6 +90,10 @@ internal static void Unknown(string key)
                         ConfigChannel = "configChannel", AbortOnConnectFail = "abortConnect", ResolveDns = "resolveDns",
                         ChannelPrefix = "channelPrefix", Proxy = "proxy", ConnectRetry = "connectRetry",
                         ConfigCheckSeconds = "configCheckSeconds", ResponseTimeout = "responseTimeout", DefaultDatabase = "defaultDatabase";
+#if !CORE_CLR
+            internal const string SslProtocols = "sslProtocols";
+#endif

```

Comment:
Same as comment above - what's specific to non-Core? The documentation fragmenting between the two is not an awesome distinction to have to make.